CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -1228,7 +1219,13 @@ if nameAndHealthMenuItem and topbarEnabled and not isTenFootInterface then
 	AddItemInOrder(RightMenubar, nameAndHealthMenuItem, RIGHT_ITEM_ORDER)
 end
 
-
+local Panel3D = require(GuiRoot.Modules.Panel3D)
```

Comment:
We have a loosely enforced standard of putting all the requires at the top unless they are needed somewhere else.